tests/foreman/api/test_contentviewfilter.py

```diff
@@ -103,6 +107,8 @@ def test_positive_create_erratum_with_name(self):
             correct name and type
 
         :CaseLevel: Integration
+
+        :CaseImportance: High
```

Comment:
Same here, we have it above https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentviewfilter.py#L17